int32 setpoint0
int32 setpoint1
int32 setpoint2
int32 setpoint3
int32 speed0
int32 speed1
int32 speed2
int32 speed3
